### PR TITLE
Add initial ProtoBuf for DevToolsMessages.

### PR DESCRIPTION
**Pull request by SHeimlich**
- Add devtools_message.proto file
- Create and send message in DevToolsService with the new ProtoBuf.

cc: @piotrswigon @jasonwyatt

### CODE REVIEW

File: java/arcs/android/devtools/DevToolsService.kt [hunk 1]
```diff
@@ -79,6 +80,9 @@ open class DevToolsService : Service() {
         object : IDevToolsProxyCallback.Stub() {
           override fun onProxyMessage(proxyMessage: ByteArray, storageKey: String) {
             scope.launch {
+              val proto = DevtoolsMessage.DevToolsMessageProto.newBuilder()
+              proto.proxyMessage = proxyMessage.decodeProxyMessage().toProto()
+              devToolsServer.send(proto.build().toString())
```
Comment (csilvestrini) [inline java/arcs/android/devtools/DevToolsService.kt:85]: Optional: you can chain this up like so:

```
val proto = DevtoolsMessage.DevToolsMessageProto.newBuilder()
  .setProxyMessage(proxyMessage.decodeProxyMessage().toProto())
  .build()
```

File: java/arcs/android/devtools/devtools_message.proto [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+package arcs.android.devtools;
+
+import "java/arcs/android/storage/proxy_message.proto";
+import "google/protobuf/wrappers.proto";
+
+option java_package = "arcs.android.devtools";
+
+message DevToolsMessageProto {
+  // ID is a nullable int32, so encode it as an Int32Value.
+  google.protobuf.Int32Value id = 1;
```
Comment (csilvestrini) [inline java/arcs/android/devtools/devtools_message.proto:12]: It doesn't look like you use this, is it actually needed?

If not, you could delete it, and then you can get rid of all that wrappers boilerplate

**Review by csilvestrini — APPROVED** [review]

File: java/arcs/android/devtools/devtools_message.proto (hunk 2 above)
Comment (SHeimlich) [reply java/arcs/android/devtools/devtools_message.proto]: Oops, I changed the structure late Friday and forgot to update the code. I've removed the id and added in the StorageKey and StoreType.

File: java/arcs/android/devtools/devtools_message.proto [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import "java/arcs/android/storage/proxy_message.proto";
+import "google/protobuf/wrappers.proto";
+
+option java_package = "arcs.android.devtools";
+
+message DevToolsProxyMessageProto {
+  arcs.android.storage.ProxyMessageProto proxy_message = 1;
+  enum StoreType {
+    DIRECT = 0;
+    REFERNCE_MODE = 1;
```
Comment (csilvestrini) [inline java/arcs/android/devtools/devtools_message.proto:14]: Spelling

File: java/arcs/android/devtools/devtools_message.proto [hunk 4]
```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package arcs.android.devtools;
+
+import "java/arcs/android/storage/proxy_message.proto";
+import "google/protobuf/wrappers.proto";
```
Comment (csilvestrini) [inline java/arcs/android/devtools/devtools_message.proto:6]: You can remove this now

File: java/arcs/android/devtools/BUILD [hunk 5]
```diff
@@ -39,3 +45,17 @@ arcs_kt_android_library(
         "//third_party/kotlin/kotlinx_coroutines",
     ],
 )
+
+proto_library(
+    name = "devtools_message_proto",
+    srcs = ["devtools_message.proto"],
+    deps = [
+        "//java/arcs/android/storage:proxy_message_proto",
+        "//third_party/java/arcs/deps:protobuf_wrappers_proto",
```
Comment (csilvestrini) [inline java/arcs/android/devtools/BUILD:54]: You can remove this now

**Review by csilvestrini — APPROVED** [review]

File: java/arcs/android/devtools/devtools_message.proto (hunk 4 above)
Comment (SHeimlich) [reply java/arcs/android/devtools/devtools_message.proto:6]: I don't think I can? The message is still a wrapper around the ProxyMessageProto, it's the first field.

**Pull request closed without merging**